### ISSUE TITLE
Color of items overriden because of .label class

### ISSUE THREAD

**Issue by davidrleonard** [PredixDev/px-context-browser#89]
When a global .label class is included on the page with basic CSS in shady DOM mode, it overrides the px-context-browser-item color.

Browser: Chrome 62 (any)
Polymer: 1.9.2
Shady DOM: true

### Existing behavior

Reproduce using this code snippet:

```
<!DOCTYPE html>
<html lang="en">
<head>
  <title>Context Browser Demo</title>

  <script src="../../webcomponentsjs/webcomponents.min.js"></script>
  <link rel="import" href="../../polymer/polymer.html"/>
  <link rel="import" href="../../px-theme/px-theme-styles.html"/>
  <link rel="import" href="../px-context-browser-trigger.html"/>
  <link rel="import" href="../px-context-browser.html"/>
</head>

<style is="custom-style" include="px-theme-styles"></style>
<style>
  body {
    margin: 0;
    padding: 30px;
  }
  .label {
    color: orange;
  }
</style>
<body>
  <template is="dom-bind">
    <px-context-browser-trigger trigger="{{openTrigger}}"></px-context-browser-trigger>
    <px-context-browser
        show-filter
        keys='{ "label": "name", "id": "_ID_", "children": "subnodes" }'
        items='[{ "_ID_": "home", "name": "Home", "subnodes" : [{ "_ID_": "child-1", "name": "Child 1" }, { "_ID_": "child-2", "name": "Child Two" }] }]'
        open-trigger="[[openTrigger]]">
    </px-context-browser>
  </template>
</section>
</html>
```

Open the context browser. It has an orange color.

![image](https://user-images.githubusercontent.com/6456757/30887515-ae16ddce-a2d0-11e7-9429-d543782a5ff5.png)

### Expected behavior

The first item should have a gray/black text color.

**Comment by Shibham** [comment]
Thanks David for all the help and troubleshooting ! :)

**Comment by davidrleonard** [comment]
Fixed in cb1d665
